cpp/src/tsne/bh_kernels.h

```diff
@@ -608,13 +577,8 @@ RepulsionKernel( /* int *restrict errd, */
                 const float *restrict radiusd_squared,
                 const int *restrict maxdepthd)
 {
-  // Return if max depth is too deep
-  // Not possible since I limited it to 32
-  // if (maxdepthd[0] > 32)
-  // {
-  //   atomicExch(errd, max_depth);
-  //   return;
-  // }
+  int limiter = 0;
+  int limiter2 = 0;
```

Comment:
Same Q for these limiters.

Reply:
Same as above :). Fixes to counteract infinite loops and also weird out of memory accesses.